generated/nirfmxpulse/nirfmxpulse.proto

```diff
@@ -627,13 +653,16 @@ message Cfg1ReferenceWaveformSplitResponse {
 message CfgDigitalEdgeTriggerRequest {
   nidevice_grpc.Session instrument = 1;
   string selector_string = 2;
-  string digital_edge_source = 3;
+  oneof digital_edge_source_enum {
```

Comment:
Same here as my comment on `nirfmxbluetooth.proto`. Either introduce the binary-breaking change and add the label to the PR or try to avoid the binary-breaking change by updating the `functions_addon.py` to manually adjust / keep the old field numbers going forward.